test/printer/samples/inline-element-with-element-mustache.html

```diff
@@ -0,0 +1,4 @@
+<a>
+    <MyIcon />
```

Comment:
How should we handle block elements inside inline elements? Should they wrap as normal?

Reply:
yeah I think wrap as normal

Reply:
Hmm, so it looks like Prettier handles this differently (with `htmlWhitespaceSensitivity` set to `strict` or `css`): even with child block tags, it doesn't break immediately after the opening tag nor immediately before the closing tag.

[Here are some examples]

Coincidentally, that behavior is the easiest way to implement the fixes for inline elements in this plugin. [Here's the updated behavior]

[Here are some examples]: https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4D4AWmqA9FnhjgXADbUQlm4A6ULqAzgA4CGUuAnvWJdeuDjz4xCUAOYkRfFmwUEijfDToNKheROaso40RTwbaQxqdXbyw-SAA0ICJxiZo7ZKG4AnXxAA7gAKfgheKNzYEJjoTiAARr7cYADWcDAAyjxgmLLIML4ArnDO+DAAttQA6oTwImBwmeGY7tit-Mjg7F7OeexwvjDByTIV3MgAZtzUA84AVuwAHgBCyWkZmdwVcAAyeXBTM3Mgi0uZeTLUcACKRRDwR7OlIDy+A75d5VXxnL55MGqsSkyAAHAAGZx-CADarJThdP5wD7YQ7OACO93gI1cERA3HYAFooHA4OgyfFfHBMZgqSNuGMJkhps9nAMKpgCsUXuxLtc7g9Dszji8YNwEkD0CCkAAmZyFbiYaiXADCEAq4y6UGgaJARQGABVxREWQMAL5moA

[Here's the updated behavior]: https://github.com/UnwrittenFun/prettier-plugin-svelte/pull/41/commits/a599244c1563a59baaac7af5a52b34a0f77c83c4